jicoco/src/main/kotlin/org/jitsi/rest/JettyHelpers.kt

```diff
@@ -68,43 +64,29 @@ fun createJettyServer(
  * client auth is needed for SSL (see [SslContextFactory.setNeedClientAuth]).
  * [sendServerVersion] controls whether Jetty should send its server version in the error responses or not.
  */
-fun createSecureJettyServer(
-    port: Int,
-    keyStorePath: String,
-    host: String? = null,
-    keyStorePassword: String? = null,
-    needClientAuth: Boolean = false,
-    sendServerVersion: Boolean = true
-): Server {
-    val sslContextFactoryKeyStoreFile = Paths.get(keyStorePath).toFile()
+fun createSecureJettyServer(config: JettyBundleActivatorConfig): Server {
+    val sslContextFactoryKeyStoreFile = Paths.get(config.keyStorePath!!).toFile()
     val sslContextFactory = SslContextFactory.Server().apply {
-        if (supportsTls13()) {
-            setIncludeProtocols("TLSv1.2", "TLSv1.3")
+        val tlsProtocols = if (supportsTls13()) {
```

Comment:
I'm not sure if we still need `supportsTls13` - I think it's a JVM 1.8 thing, which I seem to recall we're not supporting anymore for other reasons?